agent-connection/src/main/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/AgentConnection.kt

```diff
@@ -46,7 +56,15 @@ interface Connection {
     fun receiveProof(): ProofInfo
 }
 
-class AgentConnection(val myAgentUrl: String, val invite: String? = null, val userName: String = "user1", val passphrase: String = "test") : Connection {
+class AgentConnection(val myAgentUrl: String,
```

Comment:
May be we should keep old API for compatibility?
Or delete invite totally?

Reply:
It has more to do with agent IP address, which I changed. The tests should pass when python agents are executed in a local venv rather than from the docker image.

Reply:
I think that test should be able to pass when they are executed in any env, only IP`s should change
But here i`m talking about change of parameters sequence, which will result in errors in external project without any compilation errors.

Reply:
> I think that test should be able to pass when they are executed in any env
Sure. It actually passes when the IP-address is specified correctly.
> But here i'm talking about change of parameters sequence, which will result in errors in external project without any compilation errors.
As agreed, I am removing "invite" parameter from the constructor and keep the method acceptInvite(). Please note there's no such thing like Old and New interface here. We're now working on the initial version.

Reply:
If you will remove invite at all, then its totally OK, but if we were keeping it, I don't think that it is wise to swap parameters in places.

Reply:
apologies, that wasn't intentional.